### PR TITLE
Обновил ru перевод СSS Custom Properties

### PR DESCRIPTION
**Pull request by temanaganov**

### CODE REVIEW

**Comment by sashasushko** [comment]
@temanaganov thank you for your help. Sorry for long response. I remember about your PR. And I'll review it ASAP. Peace 🕊

File: files/ru/web/css/--_star_/index.md [hunk 1]
```diff
@@ -1,14 +1,88 @@
 ---
-title: Переменные (--*)
+title: 'Кастомные свойства (--*): CSS переменные'
 slug: Web/CSS/--*
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:3]:
```suggestion
slug: Web/CSS/--*
page-type: guide
```

File: files/ru/web/css/--_star_/index.md [hunk 2]
```diff
@@ -1,14 +1,88 @@
...
-  - CSS Variables
-  - Property
+  - CSS Custom Properties
+  - Guide
+  - Reference
 translation_of: Web/CSS/--*
+browser-compat: css.properties.custom-property
 ---
-{{CSSRef}}{{SeeCompatTable}}
+{{CSSRef}}
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:12]:
```suggestion

{{CSSRef}}
```

File: files/ru/web/css/--_star_/index.md [hunk 3]
```diff
@@ -1,14 +1,88 @@
...
 
-Имена свойств с префиксом `--`, типа `--example-name`, представляют собой _пользовательские свойства_, содержащие значение, которое можно использовать по всему документу, используя ({{cssxref("var()")}}).
+Имена свойств с префиксом `--`, например `--example-name`, представляют собой _кастомные свойства_, содержащие значение, которое можно использовать по всему документу, используя ({{cssxref("var()")}}).
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:14]:
```suggestion
Имена свойств с префиксом `--`, например `--example-name`, представляют собой _кастомные свойства_, содержащие значения, которые можно переиспользовать в прочих декларациях, при помощи функции ({{cssxref("var()")}}).
```

File: files/ru/web/css/--_star_/index.md [hunk 4]
```diff
@@ -1,14 +1,88 @@
...
 
-Пользовательские свойства участвуют в каскаде: каждое из них может появляться несколько раз, а значение переменной соответствует значению, определённому в пользовательском свойстве алгоритмом каскадирования.
+Кастомные свойства привязаны к элементам, для которых они объявлены, и они также участвуют в каскаде: значение такого свойства определяется каскадным алгоритмом.
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:16]:
```suggestion
Область действия кастомного свойства ограничена элементом, для которого оно объявлено. Кастомные свойства также могут менять свои значения согласно правилам каскадирования.
```

File: files/ru/web/css/--_star_/index.md [hunk 5]
```diff
@@ -1,14 +1,88 @@
...
+```css
+--somekeyword: left;
+--somecolor: #0000ff;
+--somecomplexvalue: 3px 6px rgb(20, 32, 54);
+```
+
+- `<declaration-value>`
+  - : Это значение соответствует любой последовательности из одного или нескольких токенов, если последовательность не содержит запрещенных токенов. Оно представляет собой все, что действительное объявление может иметь в качестве значения.
+
+> **Замечание:** Имена кастомных свойств чувствительны к регистру — `--my-color` будет рассмотрено, как отдельное кастомное свойство по отношению к `--My-color`.
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:31]:
```suggestion
> **Примечание:** Имена кастомных свойств чувствительны к регистру — таким образом `--my-color` и `--My-color` будут определены, как два различных кастомных свойства.
```

File: files/ru/web/css/--_star_/index.md [hunk 6]
```diff
@@ -1,14 +1,88 @@
...
+### Синтаксис
+
+{{CSSSyntax}}
+
+## Пример
+
+### HTML
+
+```html
+<p id="firstParagraph">Этот параграф должен иметь синий фон и жёлтый текст.</p>
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:42]:
```suggestion
<p id="firstParagraph">
  Этот параграф должен иметь синий фон и жёлтый текст.
</p>
```

File: files/ru/web/css/--_star_/index.md [hunk 7]
```diff
@@ -1,14 +1,88 @@
...
+<p id="secondParagraph">Этот параграф должен иметь жёлтый фон и синий текст.</p>
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:43]:
```suggestion
<p id="secondParagraph">
  Этот параграф должен иметь жёлтый фон и синий текст.
</p>
```

File: files/ru/web/css/--_star_/index.md [hunk 8]
```diff
@@ -1,14 +1,88 @@
...
+<div id="container">
+  <p id="thirdParagraph">Этот параграф должен иметь зелёный фон и жёлтый текст.</p>
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:45]:
```suggestion
  <p id="thirdParagraph">
    Этот параграф должен иметь зелёный фон и жёлтый текст.
  </p>
```

File: files/ru/web/css/--_star_/index.md [hunk 9]
```diff
@@ -1,14 +1,88 @@
...
+```
+
+### CSS
+
+```css
+:root {
+  --first-color: #16f;
+  --second-color: #ff7;
+}
+#firstParagraph {
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:56]:
```suggestion

#firstParagraph {
```

File: files/ru/web/css/--_star_/index.md [hunk 10]
```diff
@@ -1,14 +1,88 @@
...
+  background-color: var(--first-color);
+  color: var(--second-color);
+}
+#secondParagraph {
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:60]:
```suggestion

#secondParagraph {
```

File: files/ru/web/css/--_star_/index.md [hunk 11]
```diff
@@ -1,14 +1,88 @@
...
+  background-color: var(--second-color);
+  color: var(--first-color);
+}
+#container {
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:64]:
```suggestion

#container {
```

File: files/ru/web/css/--_star_/index.md [hunk 12]
```diff
@@ -1,14 +1,88 @@
...
+  --first-color: #290;
+}
+#thirdParagraph {
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:67]:
```suggestion

#thirdParagraph {
```

File: files/ru/web/css/--_star_/index.md [hunk 13]
```diff
@@ -1,14 +1,88 @@
...
+{{Specifications}}
+
+## Совместимость с браузерами
+
+{{Compat}}
+
+## Смотрите также
+
+- [Использование переменных в CSS](/ru/docs/Web/CSS/Using_CSS_custom_properties)
+- Функция {{cssxref("var", "var()")}}
```
Comment (Saionaro) [inline files/ru/web/css/--_star_/index.md:88]:
```suggestion
- Функция {{cssxref("var", "var()")}}

```

**Review by saionaro — APPROVED** [review]
@temanaganov спасибо за правки!

**Pull request merged**